src/main/java/mate/academy/Main.java

```diff
@@ -1,7 +1,21 @@
 package mate.academy;
 
+import mate.academy.dao.BookDao;
+import mate.academy.lib.Injector;
+import mate.academy.model.Book;
+
 public class Main {
+
     public static void main(String[] args) {
+        Injector injector = Injector.getInstance("mate.academy");
+        BookDao bookDao = (BookDao) injector.getInstance(BookDao.class);
+        Book book = new Book();
+        bookDao.create(book);
+        bookDao.update(book);
+        bookDao.findAll();
```

Comment:
you did not check the result of this method, need some printing to the console or something else